app/views/events/_event_card.html.erb

```diff
@@ -1,6 +1,5 @@
-<article class="event-card">
+<article class="event-card" data-tags="">
```

Comment:
might be about to utilize the  .parameterize.underscore here to populate this attribute.